common/src/main/java/com/wynntils/core/WynntilsMod.java

```diff
@@ -121,6 +122,8 @@ private static void handleExceptionInEventListener(Throwable t, Event event) {
 
         WynntilsMod.error("Exception in feature " + feature.getTranslatedName(), t);
         WynntilsMod.warn("This feature will be disabled");
+        WynntilsMod.postEvent(
+                new WynntilsCrashEvent(feature.getClass().getName(), WynntilsCrashEvent.CrashType.FEATURE, t));
```

Comment:
I just wanted to add these in the function refactor as something like `FeatureStateChangeEvent.Crashed` (disabled and enabled too..). I guess let's not forget to handle this at merge.

Reply:
Oh, so this works with overlay and screens too. I guess we might need to have some level of duplication. But that is a problem for the future.

Reply:
Yeah, it is a bit of duplication. I'm guessing we might perhaps add a `WynntilsMod.reportCrash`, if we can tease out enought commonality and still keep the reporting useful.

Are you OK with saving that for the future, or do you want me to make it in this PR?

Reply:
I did it. It was also triggered by the fact that I realized we also have crash protection for annotators.